data/repositories.toml

```diff
@@ -417,6 +417,7 @@ repositories = [
   'github.com/neovim/neovim',
   'github.com/nestjsx/crud',
   'github.com/netlify/netlify-cms',
+  'github.com/neurodata/hyppo'
```

Comment:
Looks like you're missing a comma at the end of the line

Reply:
Right, my bad, fixed in https://github.com/deepsourcelabs/good-first-issue/pull/234/commits/155c65f11b258b119e4e6d9e3bf8211b19ef2f91